src/Context.php

```diff
@@ -204,6 +210,16 @@ public function enableMaxDepthChecks(): self
         return $this;
     }
 
+    /**
+     * @return $this
+     */
+    public function enableInheritGroups(): self
+    {
+        $this->attributes['inherit_groups'] = true;
```

Comment:
It is called `inherited_groups` above, but `inherit_groups` here. Is that right?